packages/regl-worldview/src/utils/HitmapIdManager.js

```diff
@@ -0,0 +1,74 @@
+// @flow
+
+//  Copyright (c) 2018-present, GM Cruise LLC
+//
+//  This source code is licensed under the Apache License, Version 2.0,
+//  found in the LICENSE file in the root directory of this source tree.
+//  You may not use this file except in compliance with the License.
+
+import last from "lodash/last";
+import React from "react";
+
+import type { HitmapId, MouseEventObject, BaseShape } from "../types";
+
+function fillArray(start: number, length: number): Array<number> {
+  return new Array(length).fill(0).map((_, index) => start + index);
+}
+
+type CommandInstance = React.Component<any>;
+
+export default class HitmapIdManager {
+  _hitmapIdMap: { [HitmapId]: BaseShape } = {}; // map hitmapId to the original marker object
```

Comment:
Can make the code self-documenting here
```suggestion
  _objectsByHitmapId: { [HitmapId]: BaseShape } = {};
```